cmd/verify.go

```diff
@@ -39,27 +40,44 @@ func NewVerifyCmd() *cobra.Command {
 				if !repo.Disabled && repo.GPGKey != "" {
 					resp, err := http.Get(repo.GPGKey)
 					if err != nil {
-						return fmt.Errorf("could not fetch gpgkey %s: %v", repo.GPGKey, err)
+						return fmt.Errorf("could not fetch gpgkey %s: %w", repo.GPGKey, err)
 					}
 					defer resp.Body.Close()
 					keys, err := openpgp.ReadArmoredKeyRing(resp.Body)
 					if err != nil {
-						return fmt.Errorf("could not load gpgkey %s: %v", repo.GPGKey, err)
+						return fmt.Errorf("could not load gpgkey %s: %w", repo.GPGKey, err)
 					}
 					for _, k := range keys {
 						keyring = append(keyring, k)
 					}
 				}
 			}
 
-			workspace, err := bazel.LoadWorkspace(verifyopts.workspace)
-			if err != nil {
-				return fmt.Errorf("failed to open workspace %s: %v", verifyopts.workspace, err)
-			}
-			for _, rpm := range bazel.GetRPMs(workspace) {
-				err := verify(rpm, keyring)
+			if verifyopts.fromMacro == "" {
```

Comment:
I wonder if we want to automatically load macros and check for RPMs going from the workspace, but I don't think this is something which needs to be resolved in this PR. Fine with it how it is.